pages/api/org/[orgSlug]/project/index.ts

```diff
@@ -34,14 +34,21 @@ async function handler(
     if (req.method === "POST") {
       const { orgSlug } = req.query
 
-      const { data: projects, error } = await supabase
-        .from("project")
-        .select("*, organization!inner(*)")
-        .eq("organization.slug", orgSlug)
+      const { data: orgData, error: orgError } = await supabase
+        .from("organization")
+        .select("id, slug")
+        .eq("slug", orgSlug)
+        .limit(1)
+        .single()
 
-      if (error) return res.status(404).json(error)
+      if (orgError) return res.status(500).json(orgError)
+
+      const { data, error } = await supabase
+        .from("project")
+        .insert({ ...req.body, org_id: orgData.id })
```

Comment:
I know I need to get the project doing: `.insert({ ...req.body, org_id: orgData.id }).select().single()` but unfortunately, I encountered an error while attempting to do so.

```
[\new row violates row-level security policy for table "projects"]
```
I think the reason for this issue is a misconfiguration in my project or database that is preventing this validation from passing. `(is_member_org((org_id)::text) AND is_current_org((org_id)::text))`

Reply:
Im skipping the org validation: `let currentOrgExist = true`

Reply:
I think is the expected behavior. First create the project and then you can select that project. If your are chaining the calls like this `.insert({ ...req.body, org_id: orgData.id }).select().single()` supabase reject the select part because the project is not created yet so no possible to validate the org_id. I think that happend to me when creating organizations so I ended up using a postgres function.